npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater.rb

```diff
@@ -326,6 +326,20 @@ def write_lockfiles
           end
         end
 
+        def lock_deps_with_latest_reqs(content)
```

Comment:
Looks like this isn't actually needed as the tests are passing and this isn't called anywhere, which would mean yarn doesn't have this problem 🙌 

We should remove ✂️ this method and keep the yarn test (making sure this doesn't get introduced in some future version of yarn).

Reply:
Excellent!